scripts/utils/match_state.py

```diff
@@ -0,0 +1,34 @@
+import rospy
+from std_msgs.msg import Bool
+
+class MatchState:
+    def __init__(self, topic_name):
+        """Cria um novo objeto para fazer a leitura do estado da partida
+
+        Args:
+            topic_name (string): Nome do tópico do estado da partida
+        """
+        self.state = False
+        self.topic_name = topic_name
+
+    def initialise(self):
+        """Inicializa o subscriber para leitura do estado da partida
+        """
+        rospy.Subscriber(self.topic_name, Bool, self._callback)
+        rospy.loginfo(f"Inicializando leitor do estado da partida {rospy.get_time()}")
+
+    def _callback(self, data):
+        """Função de callback necessária para receber informação do subscriber
+
+        Args:
+            data (std_msgs/UInt32): Dados da leitura do estado da partida
+        """
+        self.brightness = data.data
```

Comment:
```suggestion
        self.state = data.data
```
Alguem copiou e colou ai em 👀 aushuaushua

Reply:
Kkkkkkkk nem sei do que ta falando